server/boot/controller-boot.js

```diff
@@ -0,0 +1,30 @@
+var controller;
+
+module.exports = function(app) {
+  // here app.controller(ctrl) creates a controller wrapper to register
+  // the 'ctrl' class with strong-remoting
+  app.controller(ctrl).remoteMethod('speak', {
+    http: { path: '/', verb: 'get' },
+    returns: { arg: 'msg', type: 'string', root: true },
+  });
+};
+
+function ctrl(app, ctx) {
+  // actually ends up registering at /api/ctrl
+  this.name = 'TestName';
```

Comment:
> actually ends up registering at /api/ctrl

Is this because we use `fn.name` to build the endpoint path? I think we should allow the user to arbitrarily customize the path, e.g. by registering the controller like this:

```js
app.controller(ctrl, { http: { path: '/myctrl' } });
```